ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/execution/PowBlock.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.datastructures.execution;
+
+import org.apache.tuweni.bytes.Bytes32;
+import org.apache.tuweni.units.bigints.UInt256;
+import org.web3j.protocol.core.methods.response.EthBlock;
+import tech.pegasys.teku.ssz.containers.Container4;
+import tech.pegasys.teku.ssz.containers.ContainerSchema4;
+import tech.pegasys.teku.ssz.primitive.SszBytes32;
+import tech.pegasys.teku.ssz.primitive.SszUInt256;
+import tech.pegasys.teku.ssz.schema.SszPrimitiveSchemas;
+import tech.pegasys.teku.ssz.tree.TreeNode;
+
+public class PowBlock extends Container4<PowBlock, SszBytes32, SszBytes32, SszUInt256, SszUInt256> {
```

Comment:
the only reason I did it in merge-interop was for reference tests. They provide PowBlock in ssz format.
But going back to this you're right and it do not make sense to have ssz overhead for tests.
my idea is to provide a non-ssz version with _`fromSsz`_ (and  _`fromWeb3`_ https://github.com/ConsenSys/teku/pull/4573#discussion_r743263102 if possible)

Reply:
We can just ignore the ssz_static tests for this (we already do for Eth1Block) as they add no value since none of the SSZ functionality is actually used.  So I don't think we should even need the `fromSsz`.